server/workflow-client.js

```diff
@@ -213,6 +213,63 @@ function cliTransform(item) {
   return item;
 }
 
+function getCreds() {
+  let creds;
+
+  if ('TEMPORAL_TLS_KEY_PATH' in process.env) {
+    if (!'TEMPORAL_TLS_CERT_PATH' in process.env) {
+      throw Error('TLS certificate is not provided');
+    }
+
+    const pk = process.env.TEMPORAL_TLS_KEY_PATH;
+    const cert = process.env.TEMPORAL_TLS_CERT_PATH;
+
+    let ca;
+    if ('TEMPORAL_TLS_CA_PATH' in process.env) {
+      ca = process.env.TEMPORAL_TLS_CA_PATH;
+      caContent  = fs.readFileSync(ca)
+    }
+
+    let verifyHost = false;
+    const checkServerIdentity;
+    if ('TEMPORAL_TLS_ENABLE_HOST_VERIFICATION' in process.env) {
+      verifyHost = process.env.TEMPORAL_TLS_ENABLE_HOST_VERIFICATION;
+      hostName = process.env.TEMPORAL_TLS_SERVER_NAME;
+      checkServerIdentity = (hostname, cert)=> {
+        // verify server
+      } 
+    }
+
+    if(checkServerIdentity){
```

Comment:
What's this for?

Reply:
actually it is always true so it breaks the credentials